app/models/slug_migration_publisher.rb

```diff
@@ -1,24 +1,24 @@
 require "gds_api/publishing_api_v2"
 
 class SlugMigrationPublisher
-  def process(slug_migration)
+  def process(content_id:, old_path:, new_path:)
```

Comment:
I reckon it'd be nice to rename this to `RedirectPublisher` as it's no longer about SlugMigrations?

It'd be good to use the `PUBLISHING_API` constant too. Maybe it could be a constructor argument like in `Publisher`?